src/notifications/generate.ts

```diff
@@ -318,4 +324,21 @@ export const generateNotificationMap: Record<
       .icon(NotificationIcon.Bell)
       .avatarSource(ctx.source)
       .objectPost(ctx.post, ctx.source, ctx.sharedPost),
+  squad_public_approved: (
+    builder: NotificationBuilder,
+    ctx: NotificationSourceContext,
+  ) =>
+    builder
+      .icon(NotificationIcon.DailyDev)
+      .referenceSource(ctx.source)
+      .targetSource(ctx.source)
+      .avatarSource(ctx.source),
+  squad_public_rejected: (
+    builder: NotificationBuilder,
+    ctx: NotificationSourceContext,
+  ) => builder.systemNotification().referenceSource(ctx.source),
+  squad_public_submitted: (
+    builder: NotificationBuilder,
+    ctx: NotificationSourceContext,
+  ) => builder.systemNotification().referenceSource(ctx.source),
```

Comment:
Did you test that when people request after 14 days we again get a notification, because due to how notifications v2 work I think it would just reuse the old notification and not send anything. We might have to add `request.id` as `uniqueKey`

Reply:
Right. Very good catch, yes, the reference key needs to be something unique across requests.